oca_dependencies.txt

```diff
@@ -0,0 +1,2 @@
+account-financial-tools git@github.com:vauxoo-dev/account-financial-tools.git 12.0-account_reversal-hbto
+account-closing
```

Comment:
This needs to point towards https://github.com/vauxoo-dev/account-financial-tools/commits/12.0-account_reversal-hbto

in order it to work properly as in https://git.vauxoo.com/vauxoo/enterprise-mx/merge_requests/468/diffs#03a80bdbacce644481ba04cd71187eab200677f9_3_4

and this CI can run smoothly
https://runbot.odoo-community.org/runbot/repo/github-com-oca-account-closing-89 on your PR.

Regards